src/features/modals/ConnectWalletPrompt/ConnectWalletPrompt.tsx

```diff
@@ -0,0 +1,12 @@
+//- React Imports
```

Comment:
Nice thanks. I assume we'd want to show the marketplace content if wallet is not connected, rather than what I've quickly put together below. Will fix this up as I work through the feedback on the modals below as I noticed a few changes will be made :) 

<img width="2363" alt="Screenshot 2022-09-02 at 16 17 06" src="https://user-images.githubusercontent.com/39112648/188181043-7552ad32-8494-4b0f-9009-7e5bf6d5f196.png">

Reply:
Going to resolve this conversation as I've now removed the connect wallet prompt modal I had added